client/components/inline-support-link/context-links.js

```diff
@@ -106,11 +106,11 @@ const contextLinks = {
 		blog_id: DEVELOPER_WORDPRESS_BLOG_ID,
 	},
 	'hosting-clear-cache': {
-		link: 'https://wordpress.com/support/clear-your-sites-cache/',
+		link: 'https://developer.wordpress.com/docs/site-performance/global-edge-cache/#1-clear-your-site-s-cache',
 		post_id: 164969,
 	},
 	'hosting-connect-to-ssh': {
-		link: 'https://wordpress.com/support/connect-to-ssh-on-wordpress-com/',
+		link: 'https://developer.wordpress.com/docs/developer-tools/ssh/',
 		post_id: 213309,
```

Comment:
Similar to the previous one. We may want to add `blog_id: DEVELOPER_WORDPRESS_BLOG_ID,` and change the post_id to `100385`.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/88766/commits/58092cc6df9a77d21aec91506dd51096916fbc1a